docs/src/main/asciidoc/mongodb.adoc

```diff
@@ -502,6 +502,18 @@ public class CodecFruitService {
 }
 ----
 
+== The POJO Codec
+
+Quarkus initializes the link:http://mongodb.github.io/mongo-java-driver/3.12/bson/pojos[MongoDB POJO Codec], this codec is capable of automatically working with POJOs so in many cases,
```

Comment:
I don't think this sentence provides any useful information. I propose we drop it and add the link to the next sentence

Reply:
I think we need to tell the user that the POJO codec is initialized by Quarkus in some way, and in the previous section we talk about creating custom codec so I wanted to make it clear that this is not mandatory as in most cases the pojo codec will works OOTB.

Reply:
Then perhaps the 2 sentences can be combined like so:

```
The POJO Codec provides a set of annotations that enable the customization of the way a POJO is mapped to a MongoDB collection and This codec is initialized automatically by Quarkus
```